File: MODUL-1/bahasa c/PRAK0104-231081721007-RaymondHariyono.c
#include <stdio.h>
int main(){

int hargaSepatuA;
    printf("Harga sepatu A adalah "); 
    scanf("%d" , & hargaSepatuA);
   
int hargaSepatuB;
    printf("Harga sepatu B adalah "); 
    scanf("%d" , & hargaSepatuB);

float diskonA= 0.13;
float diskonB= 0.21;

int hargadiskonA = hargaSepatuA*(1-diskonA);
int hargadiskonB = hargaSepatuB*(1-diskonB);

printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n" , hargadiskonA);
printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %d" , hargadiskonB);

return 0;
 }

File: MODUL-1/python/PRAK0104-231081721007-RaymondHariyono.py

hargaSepatuA = int(input("Harga sepatu A adalah "))
hargaSepatuB = int(input("Harga sepatu B adalah "))

diskonA = 0.13
diskonB = 0.21

hargadiskonA = hargaSepatuA - hargaSepatuA * diskonA
hargadiskonB = hargaSepatuB - hargaSepatuB * diskonB

print(f"Sepatu A mendapat diskon 13%% sehingga harganya menjadi {hargadiskonA:.0f}")
print(f"Sepatu B mendapat diskon 21%% sehingga harganya menjadi {hargadiskonB:.0f}")